src/commands/validate.js

```diff
@@ -3,9 +3,31 @@ const colors = require('colors/safe');
 
 const utils = require('../utils');
 
+// shadow of engine/style_checker/_condense_issues
```

Comment:
yeah, I went back and forth on including it. at least for us, it adds context to this strange looking method (which has a similar copy elsewhere. Good general though though